docs/modules/ROOT/pages/how-tos/prebackuppod.adoc

```diff
@@ -0,0 +1,74 @@
+= How to use Pre-Backup pods
+
+Although K8up supports xref:how-tos/application-aware-backups.adoc[executing backup commands] in already running pods, there might be a need to start a specific Pod for the backup.
+Or what if the backup command you'd like to run doesn't belong to a running pod?
+Or you want to backup something that's running on a RWO PVC?
+Or even outside the Kubernetes cluster?
+
+[source,yaml]
+----
+apiVersion: backup.appuio.ch/v1alpha1
+kind: PreBackupPod
+metadata:
+  name: mysqldump
+spec:
+  backupCommand: mysqldump -u$USER -p$PW -h $DB_HOST --all-databases
+  pod:
+    spec:
+      containers:
+        - env:
+            - name: USER
+              value: dumper
+            - name: PW
+              value: topsecret
+            - name: DB_HOST
+              value: mariadb.example.com
+          image: mariadb
+          command:
+            - 'sleep'
+            - 'infinity'
+          imagePullPolicy: Always
+          name: mysqldump
+
+----
+
+[TIP]
+.Passing environment variables
+====
+If you want to pass environment variables to the `backupCommand` you'll have to wrap them in a shell. In above prebackup pod example that would look like this:
+
+[source]
+--
+spec:
+  backupCommand: /bin/bash -c 'mysqldump -uroot -p "${MARIADB_ROOT_PASSWORD}" --all-databases'" --overwrite
+--
+
+You can also add it to any pods that are running in the namespace:
+
+[source]
+--
+via kubectl:
+kubectl -n ${YOUR_NAMESPACE} annotate pods ${YOUR_POD_NAME} "k8up.syn.tools/backupcommand=/bin/bash -c 'mysqldump -uroot -p\"\${MARIADB_ROOT_PASSWORD}\" --all-databases'" --overwrite
+
+in the manifest:
+spec:
+  serviceName: "mariadb"
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: mariadb
+      annotations:
+        k8up.syn.tools/backupcommand: /bin/bash -c 'mysqldump -uroot -p "${MARIADB_ROOT_PASSWORD}" --all-databases'
+--
+====
+
+That's the perfect use case for using PreBackup pods!
+They're Pod definitions that live in your namespaces.
+Once the Operator triggers a backup on that specific namespace it will loop through all these Pod definitions, run them and clean them up again after the backup has finished.
```

Comment:
```suggestion
Once the Operator triggers a backup for that specific namespace, then it will loop through all these Pod definitions, execute them and clean them up after the backup finished.
```